### ISSUE TITLE
List posts of a user I interacted with

### ISSUE THREAD

**Issue by utzer** [friendica/friendica#11072]
I want to be able to show all past interactions with a user, it seems to be a good point to have this selection available under /contact/nnnnn/.

I did not describe it that well, but basically as @annando put it I want to see all posts/comments of a user with my interactions like:

- Posts that had been started by this contact and we posted a comment somewhere in the thread
- Posts that had been started by this contact and we posted a comment or liked the starting post
- Posts or comments (somewhere in a thread) by this contact that we directly commented
- Posts or comments (somewhere in a thread) by this contact that we directly commented or liked

See also discussion [here](https://social.yl.ms/display/e18176ef-2061-ae41-6534-ae1081221202).

**Comment by MrPetovan** [comment]
I believe the list of possibilities were meant as distinct and not cumulative. Each individual item in the list is straightforward to implement but implementing all of them at once would be significantly more complex.

**Comment by annando** [comment]
@MrPetovan absolutely. I think that we should chose a single item out of these suggestions. I can imagine that each of these do make sense. But on the other hand it adds complexity for the user if there were several options to chose from.

**Comment by utzer** [comment]
I thought an interactions view would be good, not sure what the complexity for the user would be if it just shows all interactions?

**Comment by annando** [comment]
I think some "Show all threads where contact X and me had interacted in any way" will be extremely confusing for the users.

**Comment by utzer** [comment]
> I think some "Show all threads where contact X and me had interacted in any way" will be extremely confusing for the users.

I wonder it we could "just" output all theses posts in a view and then apply some filter to it? I mean I know that filtering would be possible, but the question is if the data available would permit to filter it again. That would act like a search function with a filtering afterwards. 

It can all be confusing for users, but I wonder if there is anyone who sees the benefit to be able to find interactions with a user, to find common ideas or such?
